packages/react-native-windows-init/src/Cli.ts

```diff
@@ -16,6 +16,8 @@ import * as chalk from 'chalk';
 // @ts-ignore
 import * as Registry from 'npm-registry';
 
+import type generateWindows from 'react-native-windows/local-cli/lib-commonjs/generate-windows';
```

Comment:
Cannot import the real thing since we need to choose the path dynamically, but type imports should be safe.